services/cfg.py

```diff
@@ -1,5 +1,33 @@
+from enum import Enum
 
 
-
-ARTIST_INDEX = 1
-LIKES_INDEX = 22
\ No newline at end of file
+class TableEnum(Enum):
+    ID = 0
+    Artist = 1
```

Comment:
Artist, Url, Track.... etc are not a class names, so either consider them as constants (in this case use upper case LIKE_SO) or consider them as vars (in this case use snake_cake)